strawberry/asgi/handlers/http_handler.py

```diff
@@ -35,13 +35,9 @@ async def handle(self, scope: Scope, receive: Receive, send: Send):
         request = Request(scope=scope, receive=receive)
         root_value = await self.get_root_value(request)
 
-        sub_response = Response(
-            content=None,
-            status_code=None,  # type: ignore
-            headers=None,
-            media_type=None,
-            background=None,
-        )
+        sub_response = Response()
+        sub_response.status_code = None  # typing: ignore
```

Comment:
```suggestion
        sub_response.status_code = None  # type: ignore
```
This should satisfy the mypy checks :)